doc/release/migration-guide-19.2.md

```diff
@@ -0,0 +1,6 @@
+# v19.2.0
+
+## Migration Guide
+
+> This is a working draft for the up-coming v19.2.0 release.
```

Comment:
It might be safeer to add a newline after the blockquote, same wiht the release notes.

Yeah, it looks like it groups them into the same paragraph